assets/src/components/editing/elements/list/listActions.ts

```diff
@@ -80,7 +138,7 @@ export const listSettings = [
       const [, at] = [...Editor.nodes(editor)][1];
       Transforms.setNodes(
         editor,
-        { type: 'ul' },
+        { type: 'ul', style: undefined },
```

Comment:
Important to clear it out so when switching from ol to ul you don't have an invalid style set.